docs/src/components/MDXContent/renderers.js

```diff
@@ -0,0 +1,157 @@
+  };
+
+  renderers.pre = (codeBlockProps: { value: string }) => {
+    const { value, ...codeBlockRest } = codeBlockProps;
+    return <BpkCodeBlock {...codeBlockRest}>{value}</BpkCodeBlock>;
+  };
+
+  TAG_NAMES.forEach(tag => {
+    renderers[tag] = (headingProps: { children: string }) => {
+      const { children, ...headingRest } = headingProps;
+
+      const flatten = (text, child) => {
+        return typeof child === 'string'
+          ? text + child
+          : React.Children.toArray(child.props.children).reduce(flatten, text);
+      };
+
+      // https://github.com/remarkjs/react-markdown/issues/69#issuecomment-289860367
+      const id = React.Children.toArray(children)
+        .reduce(flatten, '')
+        .toLowerCase()
+        .replace(/\W/g, '-');
+
+      return (
+        // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'https://github.com/Skyscanner/backpack/blob/master/decisions/flowfixme.md'.
+        <BpkText
+          id={id}
+          textStyle={TEXT_STYLES[TAG_NAMES.indexOf(tag)]}
+          tagName={tag}
+          className={getClassName(`bpkdocs-mdx-page__heading-${tag}`)}
+          {...headingRest}
+        >
+          {children}
+        </BpkText>
+      );
+    };
+  });
+
+  renderers.img = (imageProps: { alt: string, src: string }) => {
+    const { alt, src, ...imageRest } = imageProps;
+    return (
+      // Can't use BpkImage here because it requires us to know width and height.
+      // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'https://github.com/Skyscanner/backpack/blob/master/decisions/flowfixme.md'.
+      <img /* eslint-disable-line backpack/use-components */
+        alt={alt}
+        src={src}
+        className={getClassName('bpkdocs-mdx-page__image')}
+        {...imageRest}
+      />
+    );
+  };
+
+  renderers.link = BpkLink;
+  renderers.p = Paragraph;
+
+  renderers.table = (tableProps: { children: any }) => {
+    const { children, ...tableRest } = tableProps;
+    return (
+      // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'https://github.com/Skyscanner/backpack/blob/master/decisions/flowfixme.md'.
+      <BpkMobileScrollContainer
+        className={getClassName('bpkdocs-mdx-page__mobile-scroll-container')}
+        trailingIndicatorClassName={getClassName(
+          `bpkdocs-mdx-page__mobile-scroll-indicator--trailing${
+            darkBackground ? '--dark' : '--light'
+          }`,
+        )}
+        leadingIndicatorClassName={getClassName(
+          `bpkdocs-mdx-page__mobile-scroll-indicator--leading${
+            darkBackground ? '--dark' : '--light'
+          }`,
+        )}
+        {...tableRest}
+      >
+        <BpkTable className={getClassName('bpkdocs-mdx-page__table')}>
+          {children}
+        </BpkTable>
+      </BpkMobileScrollContainer>
+    );
+  };
+
+  renderers.thead = BpkTableHead;
+  renderers.tbody = BpkTableBody;
+  renderers.tr = BpkTableRow;
+
+  renderers.tableCell = (tableCellProps: {
```

Comment:
According to https://mdxjs.com/getting-started#table-of-components we should be using `th` and `td` here, not `tableCell`

Reply:
Ah must've accidently Ctrl + z that one when making changes as I remember doing it but accidently undid it